packages/next-swc/crates/next-build/src/next_app/app_entries.rs

```diff
@@ -187,31 +187,23 @@ pub async fn get_app_entries(
     let mut entries = entrypoints
         .await?
         .iter()
-        .map(|(pathname, entrypoint)| async move {
+        .map(|(_, entrypoint)| async move {
```

Comment:
Should we just store a Set instead of a Map?

Reply:
Yeah, we might want to return a set, it's being constructed as a map though to check for duplicate paths